src/sentry/runner/commands/killswitches.py

```diff
@@ -0,0 +1,121 @@
+import click
+
+from sentry.runner.decorators import configuration
+
+
+@click.group()
+def killswitches():
+    "Manage killswitches for ingestion pipeline."
+
+
+def _safe_modify(killswitch_name, modify_func):
+    from sentry import killswitches, options
+
+    option_value = options.get(killswitch_name)
+    new_option_value = modify_func(option_value)
+
+    if option_value == new_option_value:
+        click.echo("No changes!")
+        raise click.Abort()
+
+    click.echo("Before:")
+    click.echo(killswitches.print_conditions(option_value))
+    click.echo("After:")
+    click.echo(killswitches.print_conditions(new_option_value))
+
+    click.confirm("Should the changes be applied?", default=False, show_default=True, abort=True)
+    options.set(killswitch_name, new_option_value)
+
+
+@killswitches.command()
+@click.option("--killswitch", required=True)
+@click.option("--field", required=True)
+@click.option("--value", required=True)
+@configuration
+def add_condition(killswitch, field, value):
+    """
+    Add another condition to the given killswitch.
+
+    For example, starting from a blank slate.
+
+        sentry killswitches add-condition --killswitch store.load-shed-pipeline-projects --field project_id --value 42
+
+        sentry killswitches add-condition --killswitch store.load-shed-pipeline-projects --field project_id --value 43
+
+        sentry killswitches add-condition --killswitch store.load-shed-pipeline-projects --field event_type --value transactions
```

Comment:
This I don't fully understand. 

What if the first two commands run successfully, but the third one fails (for whatever reason)? That means we drop everything for those two projects, right?

What I don't really like here is that it's not... transactional? Can we consider something similar to

```
add-condition --killswitch store.load-shed-pipeline-projects -c project_id=42 -c event_type=transactions
```

...so that it happens in one command?

Reply:
you're totally right, this should be transactional. Didn't think this through.